snapshotter/pkg/dmsetup/dmsetup.go

```diff
@@ -258,6 +258,53 @@ func Version() (string, error) {
 	return dmsetup("version")
 }
 
+// DeviceStatus represents devmapper device status information
+type DeviceStatus struct {
+	Offset int64
+	Length int64
+	Target string
+	Params []string
+}
+
+// Status provides status information for devmapper device
+func Status(deviceName string) (*DeviceStatus, error) {
+	var (
+		err    error
+		status DeviceStatus
+	)
+
+	output, err := dmsetup("status", deviceName)
```

Comment:
Do you need "--noflush"?

Reply:
Discussed with @mxpv offline.  We're both not sure whether we want `--noflush`, but we can leave it off for now.